src/plugins/mini/Examples/read.ini

```diff
@@ -0,0 +1,15 @@
+keyWithoutLeadingWhitespace = 	valueWithLeadingWhiteSpace
```

Comment:
I moved the files to the main plugin folder. I hope I never have to name a directory just like its parent folder, since I think using the same name for a subfolder is `/pretty useless/pretty useless/pretty useless`.